ruby/red-plasma/lib/plasma/client.rb

```diff
@@ -0,0 +1,9 @@
+module Plasma
+  class Client
+    alias_method :initialize_raw, :initialize
+    def initialize(socket_path)
+      socket_path = socket_path.to_path if socket_path.respond_to?(:to_path)
```

Comment:
GObject Introspection has ``(type filename)` annotation. We'll be able to remove this customization when we put the annotation to Plasma GLib and improve gobject-introspection gem to call `#to_path` for "filename" type argument automatically.

See also:

  * https://wiki.gnome.org/Projects/GObjectIntrospection/Annotations/
  * https://gitlab.gnome.org/GNOME/gtk/blob/master/gtk/gtkaccelmap.c#L666

Reply:
We can work on this later.

Reply:
I see. I'll try to work on this later.